reference-data-proxy/src/cronJobs/eStoreDealFolderCreationJob.cron.ts

```diff
@@ -6,7 +6,7 @@ import { ESTORE_CRON_STATUS, UKEF_ID } from '../constants';
 import { eStoreCronJobManager } from './eStoreCronJobManager';
 import { eStoreFacilityFolderCreationJob } from './eStoreFacilityFolderCreationJob.cron';
 import { createDealFolder } from '../v1/controllers/estore/eStoreApi';
-const facilityCreationTimer = '30 * * * * *';
+const facilityCreationTimer = '59 * * * * *';
 
 export const eStoreDealFolderCreationJob = async (eStoreData: Estore) => {
```

Comment:
Wrap this function in `try-catch` please.

Reply:
I've added this now